applications/FluidDynamicsApplication/custom_strategies/strategies/compressible_navier_stokes_explicit_solving_strategy_runge_kutta.h

```diff
@@ -407,10 +471,11 @@ class CompressibleNavierStokesExplicitSolvingStrategyRungeKutta
     ///@{
 
     bool mShockCapturing = true;
+    ShockCapturingFactoryType mShockCapturingFactory;
```

Comment:
Why you need to save the factory?

Reply:
I wanted to keep the stratgey behaviour as similar to what it was before the PR as possible. But yeah, it's unnecessary.

Reply:
Fixed